app/scripts/src/content/content-controller.js

```diff
@@ -31,8 +31,8 @@ ContentController.prototype = {
   },
 
   onSearchError: function(status, response) {
-    this.sendAnalyticsEvent({ name: 'onSearchError', value: status });
-    console.error('traktflix: Search error', status, response);
+    this.sendAnalyticsEvent({ name: 'onSearchError', value: status + (this.item && this.item.title) ? this.item.title : "Invalid item title" });
```

Comment:
Agreed. I just think that `item.title` can be `undefined` in some cases, but if `item` is `undefined`, something went very wrong and the code would fail before the search phase.
But if you prefer to leave this condition, that's ok, I just ask to put in a variable, then you can use on the `console.log` call below.